solr/core/src/java/org/apache/solr/update/UpdateShardHandler.java

```diff
@@ -138,19 +138,19 @@ public UpdateShardHandler(UpdateShardHandlerConfig cfg) {
         HttpClientUtil.createClient(
             clientParams, defaultConnectionManager, false, httpRequestExecutor);
 
+    Set<String> queryParams = new HashSet<>(2);
+    queryParams.add(DistributedUpdateProcessor.DISTRIB_FROM);
+    queryParams.add(DistributingUpdateProcessorFactory.DISTRIB_UPDATE_PARAM);
```

Comment:
Could this block be queryParams = Set.of(...);

Reply:
it can be!   I think there are a lot of places we could do that...